scripts/bump_cairo_bindings.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+git submodule update --recursive --remote --init cairo
+git add ./cairo
+git commit -m "Bump cairo bindings" --no-verify
```

Comment:
Can we include update info in README

Reply:
Yes i'm doing that, and posting on a channel. Script is not finished yet though, i will add a check if the working tree is clean (so we don't commit any more changes that we don't want)